backend/application/syson-application-configuration/src/main/java/org/eclipse/syson/application/services/DetailsViewService.java

```diff
@@ -231,4 +235,62 @@ public Element handleReferenceWidgetNewValue(Element element, String eStructural
         }
         return element;
     }
+
+    public AcceptActionUsage getAcceptActionUsage(Element self) {
+        if (self instanceof AcceptActionUsage accept) {
+            return accept;
+        }
+        return null;
+    }
+
+    public Element getAcceptActionUsagePayloadFeatureTyping(AcceptActionUsage acceptActionUsage) {
+        var payloadParam = acceptActionUsage.getPayloadParameter();
+        return payloadParam.getOwnedRelationship().stream()
+                .filter(FeatureTyping.class::isInstance)
+                .map(FeatureTyping.class::cast)
+                .findFirst()
+                .orElse(null);
+    }
+
+    public boolean setAcceptActionUsagePayloadParameter(AcceptActionUsage acceptActionUsage, Element newPayloadParameter) {
+        if (newPayloadParameter instanceof Type newType) {
+            var payloadParam = acceptActionUsage.getPayloadParameter();
+            var ft = payloadParam.getOwnedRelationship().stream()
+                    .filter(FeatureTyping.class::isInstance)
+                    .map(FeatureTyping.class::cast)
+                    .findFirst()
+                    .orElse(null);
+            if (ft != null) {
+                ft.setType(newType);
+                return true;
+            }
+        }
+        return false;
+    }
+
+    public Element getAcceptActionUsageReceiverMembership(AcceptActionUsage acceptActionUsage) {
+        var receiverArgument = acceptActionUsage.getReceiverArgument();
+        if (receiverArgument != null) {
+            return receiverArgument.getOwnedRelationship().stream()
+                .filter(Membership.class::isInstance)
+                .map(Membership.class::cast)
+                .findFirst()
+                .orElse(null);
+        }
+        return null;
+    }
+
+    public boolean setAcceptActionUsageReceiverArgument(AcceptActionUsage acceptActionUsage, Element newReceiverArgument) {
+        var receiveArgument = acceptActionUsage.getReceiverArgument();
+        var m = receiveArgument.getOwnedRelationship().stream()
```

Comment:
You should ensure that `receiveArgument` is not null before accessing `getOwnedRelationship`

Reply:
Done.